fledge-tester-list.md

```diff
@@ -0,0 +1,41 @@
+# FLEDGE Tester List
+
+## Ecosystem Testing of FLEDGE
+
+The purpose of this page is to consolidate testing information which is currently distributed across various GitHub issues, company blogs, social posts, etc.
+The usefulness of this page depends on testers sharing information and updates; if testers do not contribute or the information becomes stale it would make sense to deprecate.
+
+## Disclaimers
+
+- Not a complete list.  Testers are strongly encouraged to share their activities and insights publicly for the benefit of the broader community, but sharing is voluntary and therefore this page is not expected to reflect all testing activity.
+
+- Not evaluative. The purpose of this page is to consolidate links to information published by Privacy Sandbox testers.  Editors will review submissions for relevance and to ensure general conformance to the guidelines above, but are not evaluating or endorsing the information provided.
+
+- Editors will regularly review and approve submissions that meet the guidelines below. If you believe that an error has been submitted, please create an issue in the FLEDGE repository with the words ‘[Tester List]‘ in the subject and the Editors will respond in short order.
+
+## Guidelines
+
+- Enter information on behalf of your own organization.
+
+- Do not share test results or other detailed information inline; instead link to GitHub issues or other public pages for elaboration and discussion.
+
+- Table fields:
+    - Company/Party: The organization directly conducting tests or analysis.
+    - Est. Testing Timeframe: Expected start and duration of tests, if known.
+    - [Optional] Link to Testing Plan and/or Learnings: Link to GitHub issues, blog posts or other public information.  This could include plans for upcoming tests, or insights and summaries from completed tests.
+    - [Optional] How to Contact You: For example, a website form or reply on a GitHub issue.
+
+## How to submit
+
+1. On the FLEDGE GitHub page, navigate to the document in the main table called ‘FLEDGE-tester-list.md’
```

Comment:
"Pro tip: if you can read this sentence then you have already completed this step!" :-)

Reply:
Clarified!